includes/formFactory/ManageWikiFormFactoryBuilder.php

```diff
@@ -363,6 +363,16 @@ private static function buildDescriptorSettings(
 					$help .= "<br />{$requiresLabel}: " . implode( ' & ', $requires );
 				}
 
+				// Hack to prevent "implicit submission". See T275588 for more
+				if ( $set['type'] === 'integers' ) {
```

Comment:
Can this be `$configs['type'] === 'cloner'` to support it on other future types using cloner potentially?

Reply:
Sounds reasonable. Done!

Reply:
Thanks!